### ISSUE TITLE
All PyTorch Implementation of EfficientDet! 

### ISSUE THREAD

**Issue by innat** [rwightman/efficientdet-pytorch#94]
Hi, @rwightman hope you're doing well. 

Thanks again for your great contribution. Anyway, I've found other open-source `PyTorch` implementation of **EfficientDet**, namely:

- [Yet-Another-EfficientDet-Pytorch](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch)
- [EfficientDet.Pytorch](https://github.com/toandaominh1997/EfficientDet.Pytorch)

I didn't try on them yet and currently, I'm working on your implementation. Would you please inform me what are the differences between these implementations with respect to detection accuracy and some other essentials? Please note, I do not mean to underestimate to any, all implementations should be OK to go with.  

From your `README.md`:

> There are other PyTorch implementations. Either their approach didn't fit my aim to correctly reproduce the Tensorflow models (but with a PyTorch feel and flexibility) or they cannot come close to replicating MS COCO training from scratch.

Would you please elaborate on this?

**Comment by rwightman** [comment]
@innat What you want to use ultimately depends on your preference for style, etc. Most of the other PyTorch impl were not correct. With the exception of 'Yet Another..', which does match the original TF impl enough to load weights ported from the Google trained models like here. 

Otherwise the style of the code is quite different and I've not yet seen any reproduction of training from scratch results there that can come close to the original TPU trained models.

This version lets you create model configs with backbones from any of the models in my timm package. The bifpn and heads are based on config with less hard coded layout, similar to the original and lets you experiment with different layer types and bifpn arrangements. 

The branch I'm working on now https://github.com/rwightman/efficientdet-pytorch/tree/more_datasets .. has support for COCO, Pascal VOC, and OpenImages datasets with better support for fine-tuning (I've tested finetuning from coco to VOC on it for verification and it works very well).
